app/privkeylock/privkeylock.go

```diff
@@ -82,6 +90,11 @@ func (h Service) Run(ctx context.Context) error {
 	}
 }
 
+// Done waits until Service has finished deleting the private key lock file.
+func (h *Service) Done() {
```

Comment:
nit: any reason why the receiver is named `h` instead of `s`?

Reply:
no! thanks for pointing that out